p2p/conn/secret_connection.go

```diff
@@ -280,16 +290,19 @@ func (sc *SecretConnection) Read(data []byte) (n int, err error) {
 }
 
 // Implements net.Conn.
-func (sc *SecretConnection) Close() error                  { return sc.conn.Close() }
```

Comment:
This changes only derive from the renaming `conn` -> `underlyingConn`. They don't change behaviour.